docs/extensions/readme.md

```diff
@@ -26,6 +26,7 @@ The following documents describes AutoRest specific vendor extensions for [OpenA
 - [x-ms-long-running-operation](#x-ms-long-running-operation) - indicates that the operation implemented Long Running Operation pattern as defined by the [Resource Manager API](https://msdn.microsoft.com/en-us/library/azure/dn790568.aspx).
 - [x-nullable](#x-nullable) - when `true`, specifies that `null` is a valid value for the associated schema
 - [x-ms-header-collection-prefix](#x-ms-header-collection-prefix) - Handle collections of arbitrary headers by distinguishing them with a specified prefix.
+- [x-ms-internal](#x-ms-internal) - allow specifc operations not be exposed to users in generated clients
```

Comment:
Is there no code change to support this extension? If not, then we should probably move this down into the section for "Metadata extensions".

Reply:
@mikekistler we do generate the method as private, which allows customization by wrapping the method

Reply:
> is that just pass through by m4 and all generators read it by convention?(m4 doesn't do anything special to make it a known extension?) I do vaguely remember some conversation on standardizing it but not sure where that eneded up.

For TSP, we have a logic that label all models only referred by internal operation to internal as well. I don't know if it is also needed for swagger.